InWorldz/InWorldz.ApplicationPlugins/ChatFilter/ChatFilterModule.cs

```diff
@@ -132,7 +132,7 @@ public void AddRegion(Scene scene)
         void EventManager_OnChatFromClient(object sender, OSChatMessage chat)
         {
             if (!m_enabled) return;
-            if (chat.Message == "" || chat.SenderUUID == chat.DestinationUUID) return;
+            if (String.IsNullOrEmpty(chat.Message) || chat.SenderUUID == chat.DestinationUUID) return;
```

Comment:
Semantic change: null chat.Message now no long would cause a crash on the next line.  Likelyhood is that nulls never make it down this pipe.